test/com/amazon/ionelement/demos/kotlin/AnyElementAccessorsDemo.kt

```diff
@@ -51,28 +51,28 @@ class IonElementExtractionInKotlinTests {
                             })
                     }
                 }
-        }.asSequence().toList()
+        }.toList()
 
         assertTestCaseExtraction(stockItems)
     }
+}
 
-    private fun assertTestCaseExtraction(stockItems: List<StockItem>) {
-        val expectedTestCases = listOf(
-            StockItem(
-                "Fantastic Widget", Decimal.valueOf("12.34"), 2,
-                listOf(
-                    Order(123, "WA"),
-                    Order(456, "HI")
-                )),
-            StockItem(
-                "<unknown name>", Decimal.valueOf("23.45"), 20,
-                listOf(
-                    Order(234, "VA"),
-                    Order(567, "MI")
-                )))
 
+internal fun assertTestCaseExtraction(stockItems: List<StockItem>) {
```

Comment:
Is there a reason this was moved out of the `AnyElementAccessorsDemo` class?

Reply:
no good one, reverted.